docs/concepts/operator.md

```diff
@@ -84,6 +86,7 @@ Please refer to [this document](../usage/shoot_credentials_rotation.md#gardener-
 
 - certificate authorities (and related server and client certificates)
 - ETCD encryption key
+- `Observability` password For Plutono
```

Comment:
```suggestion
- observability passwords for Plutono
```

like in https://github.com/gardener/gardener/blob/master/docs/usage/shoot_credentials_rotation.md#gardener-provided-credentials